internal/cmd/root.go

```diff
@@ -24,14 +29,30 @@ operators available for installation, and install and uninstall
 operators from the installed catalogs.`,
 	}
 
-	flags := cmd.PersistentFlags()
+	var (
+		cfg     action.Configuration
+		timeout time.Duration
+		cancel  context.CancelFunc
+	)
 
-	var cfg action.Configuration
+	flags := cmd.PersistentFlags()
 	cfg.BindFlags(flags)
+	flags.DurationVar(&timeout, "timeout", 1*time.Minute, "The amount of time to wait before giving up on an operation.")
```

Comment:
I like how by default this is equal to the same amount of time we now retry installplans. So there's some consistency in OLM behavior across projects. cc @benluddy